### PR TITLE
refactor: permit emotions other than frustration

### PR DESCRIPTION
**Pull request by lol768**
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have to be frustrated to fill out the new feature template.

**Describe the solution you'd like**
Availability of other emotions.

**Describe alternatives you've considered**
Frustration and nothing else.

**Additional context**
No.

### CODE REVIEW

**Comment by jkcclemens** [comment]
Did you know that there's a key on your keyboard to can remove text?

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ about: Suggest an idea for this project
 ---
 
 **Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...], I'm overjoyed when [...]
```
Comment (Zarthus) [inline .github/ISSUE_TEMPLATE/Feature_request.md]:
```suggestion
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...], I'm overjoyed when [...], It mildly inconveniences me when [...], I don't feel strongly about it, but when I [...]
```

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ about: Suggest an idea for this project
 ---
 
 **Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...], I'm overjoyed when [...], It mildly inconveniences me when [...], I don't feel strongly about it, but when I [...]
```
Comment (lol768) [inline .github/ISSUE_TEMPLATE/Feature_request.md]:
```suggestion
 A clear and concise description of what the problem is. Ex. I'm always frustrated when [...], I'm underjoyed when [...], It mildly inconveniences me when [...], I don't feel strongly about it, but when I [...] 
```

Comment (lol768) [reply .github/ISSUE_TEMPLATE/Feature_request.md]: >Usually reporting a bug or missing a feature doesn't involve being overjoyed.

**Comment by jkcclemens** [comment]
Yeah

**Pull request closed without merging**